mods/sp/rules/support.yaml

```diff
@@ -1388,30 +1388,29 @@ NAWAST:
 		PipCount: 15
 		FullSequence: pip-red
 
-MUMINE:
-	Inherits: ^IWantToRunButICanMove
+CABMUMINE:
```

Comment:
We do not change code names after units are in, atleast not until there is an easy way to change this name designation for all the maps that use it.

Reply:
However there is an unused asset with the same name in zunused assets.yaml,  those two are conflicts with each other. I highly suggest if we don't change the code name here,  we should delete the one in the zunused assets.yaml to avoid conflicts.

Reply:
also,  it already in used after all

Reply:
Done fixing.